blog/content/post/2020-07-27-coreboot_blogpost.md

```diff
@@ -0,0 +1,111 @@
+being open source provides full insight of it's intestines and by that enhancing
+security. All these features make users trust coreboot more than BIOS provided
+by hardware vendors.
+
+> Do as much as needed, then jump straight payload.
+>
+> – coreboot.
+
+
+#### Why worthy
+
+The training will give you an insight
+into coreboot firmware platform, defining basics of platform boot process. You
+will learn how to build it's image and gain practical knowledge about
+customization, project structure and a source code. What about practice? We will
+show you how to reprogram SPI flash with only built coreboot binary file.
+coreboot miniseries will start on our blogpost right next week, but if you are
+interested in the full-time course, take advantage of our pre-sale and get 30%
+discount for coreboot training fundamentals.
+
+#### Why with Us
+
+If you are reading our blogposts on regular basis, you have presumably known
+most of our team or at least it's vast embedded knowledge. Our developers are
+not only knitting high quality firmware, they are also likely to share their
+experience by developing Embedded Firmware Training Program. We've been through
+a long way developing and working on coreboot. It is a high time, to pass on our
+experience in this field to support and shorten a long path of gaining
+proficiency with coreboot. Even more eagerly, having in mind that there is no
+reliable coreboot trainig available at all.
+
+![graf2](/blog/static/img/program_tree.png)
+
+#### training overview
+
+* x86 architecture
+* boot flow from power on to system take off
+* coreboot building environment
+* coreboot's customized built
+* SPI flash theory of operation
+* flashing tools and methods
+* coreboot's project structure
+* move around in project - apply changes
+* hands-on workshop using previously gained knowledge
+
+#### I am an embedded engineer. What is your superpower?
+A path of Embedded Firmware Engineering is not an easy one to follow. It not
+only forces the constant need of self-development but demands being up-to-date
+with rapid changes around both; hardware and firmware area. However, gained
+knowledge gets more and more value over time what provides stable work position
+with above-average salary and greater recognition inside the industry. Embedded engineer is a
+kind of a hybrid that brings together electrical engineering and logic skills
+of a software programming. His path brings lots of opportunities both
+in commercial and open ecosystem and even greater satisfacion thanks to the role
+of mediator between hardware and software worlds. Embedded engineers can be
+proudly called the alchemists of computing industry.
+
+If you think we can help in improving the security of your firmware or you are
+looking for a training that can boost your team of developers inside UEFI, coreboot or yocto area, feel free to [book a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```

Comment:
Yocto is written in captial letter. If this is a bug in the template please fix it.

Reply:
done